libs/api-mocks/msw/handlers.ts

```diff
@@ -76,8 +76,8 @@ export const handlers = makeHandlers({
   projectUpdate({ body, path }) {
     const project = lookup.project({ ...path })
     if (body.name) {
-      project.name = body.name
       errIfExists(db.projects, { name: body.name })
+      project.name = body.name
```

Comment:
We were updating the name in the DB and _then_ erroring if it exists, which means you could never update the name, because updating it makes it exist. Bahaha.